src/components/method-common.js

```diff
@@ -24,18 +30,20 @@ export const submitMethod = async (signal, onDismiss, method) => {
   }
 }
 
-export const reconstructToRawUrl = (url, onDismiss) => {
-  // mapping of searchValues (key) and their replaceValue (value)
-  const mapObj = {
-    github: 'raw.githubusercontent',
-    'blob/': ''
-  }
-  try {
-    url = url.replace(/\b(?:github|blob\/)\b/gi, matched => mapObj[matched])
-  } catch (error) {
-    notify('error', 'Error creating new method', { detail: (error instanceof Response ? error.text() : error) })
-    onDismiss()
-  }
-
-  return url
+export const convertToRawUrl = (methodPath, methodVersion, methodSource) => {
+  return Utils.cond(
+    // 3 Covid-19 workflows have 'Github' as source hence we check for that here to maintain backwards compatibility
+    [methodSource === MethodSource.GitHub || methodSource === 'Github', () => {
```

Comment:
(I'm not adamant about this)

To avoid having to hardcode a particular capitalization of "github", I suggest making this check case-insensitive. The safest way to do that is probably something like:

`methodSource.toLowerCase() === MethodSource.GitHub.toLowerCase()`

Reply:
Yeah that's a good point. I will make the checking case-insentitive